charts/px-monitor/templates/cortex/cortex-proxy.yaml

```diff
@@ -0,0 +1,135 @@
+{{/* Setting defaults if they are omitted. */}}
+{{- $pxMonitorEnabled := .Values.pxmonitor.enabled | default false }}
+{{- if eq $pxMonitorEnabled true }}
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: pxcentral-cortex-nginx
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: cortex-nginx
+    cortex: nginx
+{{- include "px-monitor.labels" . | nindent 4 }}
+data:
+  nginx.conf: |-
```

Comment:
Can we read this from a file and use https://helm.sh/docs/chart_template_guide/accessing_files/

Reply:
This is not a static file and needs to updated based on inputs. so can't add and load from files directory.